recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+pytorch_version:
```

Comment:
Again, it would be good to pull in the migrator. That will help keep things in sync in the future as we update pytorch.

Reply:
Sounds good, I made all of these PRs before I learned about the migrator. This package has a lot more build variants (judging by the CI), do I still only need to copy that one file you mentioned here: https://github.com/conda-forge/pytorch_geometric-feedstock/pull/20#discussion_r798133329